src/core/data-tests.js

```diff
@@ -23,11 +23,34 @@ export const name = "core/data-tests";
 const lang = defaultLang in l10n ? defaultLang : "en";
 
 function toListItem(href) {
-  return hyperHTML.bind(document.createElement("li"))`
+  
+  let modifiedHref = href.split("/").pop().split(".html")[0];
```

Comment:
nit:
```js
const [modifiedHref] = href.split("/").pop().split(".html");
```

Also, might be worth checking if the tests always end with ".html"... Some might end with just ".htm", but double check in the web platform test repository.